### PR TITLE
Dp/bugfix

### PR DESCRIPTION
**Pull request by dominicbporter**
Fixed issues and squashed bugs

### CODE REVIEW

File: api/delays.go [hunk 1]
```diff
@@ -49,3 +50,75 @@ func (e *Env) GetDelays(w http.ResponseWriter, r *http.Request) {
...
+	subs, err := e.Subscriptions.GetAll(tk.UserID)
+
+	if err != nil {
+		log.Println(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(output.JSON500Response))
+		return
+	}
+
+	var osubs []delays.OutTrip
```
Comment (haydenwoodhead) [inline api/delays.go]: I think this variable `osubs` name needs changing to something clearer. Maybe something like `subedDelays`

File: api/delays.go [hunk 2]
```diff
@@ -49,3 +50,75 @@ func (e *Env) GetDelays(w http.ResponseWriter, r *http.Request) {
...
+
```
Comment (haydenwoodhead) [inline api/delays.go]: Maybe add a comment to explain this loop. It's purpose isn't immediately obvious

**Review by haydenwoodhead — CHANGES_REQUESTED** [review]
Couple of little things I think need changing

File: api/delays.go (hunk 1 above)
Comment (haydenwoodhead) [reply api/delays.go]: Not saying `subedDelays` is the correct name just that I don't think `osubs` is the correct one.

**Comment by dominicbporter** [comment]
Have fixed issues from @haydenwoodhead

**Review by haydenwoodhead — APPROVED** [review]

**Pull request merged**